versionedRouter.js

```diff
@@ -307,32 +307,54 @@ async function handleValidation(ctx) {
 }
 
 async function routerHandleDest(ctx) {
-  const { destType, input } = ctx.request.body;
-  const routerDestHandler = getDestHandler("v0", destType);
-  if (!routerDestHandler || !routerDestHandler.processRouterDest) {
-    ctx.status = 404;
-    ctx.body = `${destType} doesn't support router transform`;
-    return null;
-  }
   const respEvents = [];
-  const allDestEvents = _.groupBy(input, event => event.destination.ID);
-  await Promise.all(
-    Object.entries(allDestEvents).map(async ([destID, desInput]) => {
-      desInput = processDynamicConfig(desInput, "router");
-      const listOutput = await routerDestHandler.processRouterDest(desInput);
-      respEvents.push(...listOutput);
-    })
-  );
-  respEvents
-    .filter(
-      resp =>
-        "error" in resp &&
-        _.isObject(resp.statTags) &&
-        !_.isEmpty(resp.statTags)
-    )
-    .forEach(resp => {
-      set(resp, "statTags.errorAt", TRANSFORMER_METRIC.ERROR_AT.RT);
-    });
+  let destType;
```

Comment:
I am also making changes this same code block so might have to resolve conflicts.

Reply:
Addressed.